pyverbs/mr.pyx

```diff
@@ -95,15 +100,19 @@ cdef class MR(PyverbsCM):
             memset(self.buf, 0, length)
         if isinstance(creator, PD):
             pd = <PD>creator
-            if implicit:
+            if null_mr:
+                self.mr = v.ibv_alloc_null_mr(pd.pd)
+            elif implicit:
                 self.mr = v.ibv_reg_mr(pd.pd, NULL, SIZE_MAX, access)
             else:
                 self.mr = v.ibv_reg_mr(pd.pd, self.buf, length, access)
             self.pd = pd
             pd.add_ref(self)
         elif isinstance(creator, CMID):
             cmid = <CMID>creator
-            if access == e.IBV_ACCESS_LOCAL_WRITE:
+            if null_mr:
+                self.mr = NULL
```

Comment:
I think it's better to just remove this check, and add an explanation to the docstring, something like "Allocate a null memory region (valid only with PD creator)"